include/extractor/extraction_way.hpp

```diff
@@ -13,6 +13,21 @@ namespace osrm
 {
 namespace extractor
 {
+namespace detail
+{
+    inline void maybeSetString(std::string &str, const char* value)
+    {
+        if (value == nullptr)
```

Comment:
personal preference here, but I'd prefer if the string should be in a valid state before getting into this function. But nothing that has to change.